packages/beacon-node/src/network/global.ts

```diff
@@ -0,0 +1,42 @@
+import { BeaconConfig, ChainForkConfig } from "@lodestar/config";
+import { NodeId, computeNodeId } from "./subnets";
+import { CustodyConfig, computeCustodyConfig } from "../util/dataColumns";
+import { PeerId } from "@libp2p/interface";
+
+/**
+ * Store shared data for different modules in the network stack.
+ * TODO: consider moving similar shared data, for example PeersData, under NetworkGlobal.
+ */
+export class NetworkGlobal {
```

Comment:
Do you mind if we name this `NetworkConfig`?   Or something similar?

Reply:
sounds good to me 👍